app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::GreetingsController < ApplicationController
```

Comment:
https://github.com/aneallaryea100/hello-rails-react/blob/ea629a87b6d24c62caf6547cbb7d508057a5dbe9/app/javascript/application.js#L1-L11

- [ ] It will be great to have your app component use routes from react-router. This is a major requirement for this project to be approved. ⬇️ 

![Screenshot 2022-11-29 at 12 44 21](https://user-images.githubusercontent.com/32361963/204494979-44df610a-6db8-4bec-8b9d-4de721f7cde5.png)

Reply:
App Component is created